### ISSUE TITLE
Syntax colour named argument names (and parameter names) differently

### ISSUE THREAD

**Issue by tombentley** [ceylon/ceylon-ide-eclipse#1634]
I find it hard to parse a named argument invocation formatted on a single line

    assertEquals { actual = scope.find("Scope"); expected = `class Scope`; };

But I think it would be easier if I could pick out the argument names as being a different colour.

**Comment by gavinking** [comment]
Actually I'm not sure I like this anymore. Consider:

```ceylon
class Person(String name) {
    print(name);
}
```

It seems to me that both occurrences of `name` should have the same color. But I need to typecheck the code to do that.

**Comment by simonthum** [comment]
I'd be on boat, provided that only named argument /invocations/ are colored differently. This is also pretty much the only place where they are being written out but are not introduced as names.
